### PR TITLE
Add example of starting controllers with launch files

### PR DESCRIPTION
**Pull request by christophfroehlich**
Rewrite of #66

### CODE REVIEW

File: example_1/doc/userdoc.rst [hunk 1]
```diff
@@ -120,46 +120,62 @@ Tutorial steps
...
+   a. Load the controller manually by
 
-    ros2 control list_controllers
+      .. code-block:: shell
 
-   what shows you that the controller is loaded but unconfigured.
+        ros2 control load_controller joint_trajectory_position_controller
 
-   .. code-block:: shell
+      what should return ``Successfully loaded controller joint_trajectory_position_controller``. Check the status
```
Comment (destogl) [inline example_1/doc/userdoc.rst:137]:
```suggestion
      what should return ``Successfully loaded controller joint_trajectory_position_controller``.
      Then check controllers' status
```

File: example_1/doc/userdoc.rst [hunk 2]
```diff
@@ -120,46 +120,62 @@ Tutorial steps
...
-    joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
-    forward_position_controller[forward_command_controller/ForwardCommandController] active
-    joint_trajectory_position_controller[joint_trajectory_controller/JointTrajectoryController] unconfigured
+      .. code-block:: shell
 
-   Configure the controller by setting it ``inactive`` by
+        ros2 control list_controllers
 
-   .. code-block:: shell
+      what shows you that the controller is loaded but unconfigured.
```
Comment (destogl) [inline example_1/doc/userdoc.rst:143]:
```suggestion
      what shows you that the controller is loaded but unconfigured. Similar output is expected:
```

**Review by destogl — CHANGES_REQUESTED** [review]
I like the docs extension and showing how controllers are managed. Nevertheless, I am not convinced that we need this launch file. We have great CLI options, and they can do everything required, so what is the added value into having this launch file.

@v-lopez if you are still there, please tell us.

Still, @bmagyar and @christophfroehlich, what do you think about his additional option? Is there something I don't see?

**Comment by christophfroehlich** [comment]
The diff view is broken here, this PR actually only adds literal c in line 167. The rest is just restructuring of the section.

I found the old PR interesting and merged it to the new structure. I thought it could be helpful for switching controller from some external interface like a GUI, or probably shows how to integrate `controller_manager.launch_utils` to a python state machine script. But because it only loads the controller but does not activate it, I agree that the benefit of this launch file is close to zero.

**Review by olivier-stasse — APPROVED** [review]
For a first example I believe that the most simple example is preferable.
We can give a try but I think most of the feedback we had was that simpler is better.

**Comment by christophfroehlich** [comment]
I don't have any hard feelings about this PR. If you think this is not adding any value to the demos but only making them more verbose -> I'll close it.

**Pull request closed without merging**